aws/elastic-beanstalk/cloudformation/beanstalk.yaml.tmpl

```diff
@@ -0,0 +1,187 @@
+AWSTemplateFormatVersion: 2010-09-09
+Description: Elasticbeanstalk test template
+Parameters:
+  BeanstalkService:
+    Type: String
+  Ec2Service:
+    Type: String
+  Partition:
+    Type: String
+  SolutionStackName:
+    Type: String
+
+Resources:
+
+  # ----------------------------------------------------------------------------
+  Application:
+    Type: AWS::ElasticBeanstalk::Application
+    Properties:
+      ApplicationVersions:
+        - Description: Version 1.0
+          SourceBundle:
+            S3Bucket: {{.S3BucketName}}
+            S3Key: python-v2.zip
```

Comment:
is the S3Key correct? in anycase I think making it a Parameter will make this template more reusable

Reply:
I had a zip within a zip. But looks like I should use `aws.s3.File` instead of `aws.s3.Folder`